types/webostv/index.d.ts

```diff
@@ -0,0 +1,1143 @@
+// Type definitions for webostv.js 1.2
+// Project: https://webostv.developer.lge.com/api/webostvjs/intro-webostvjs/
+// Definitions by: Denis Seleznev <https://github.com/hcodes>
+//                 Ilya Istomin <https://github.com/procot>
```

Comment:
you will have to ask Ilya for explicit ack his handle usage here or remove from code owners. Owners here are expected to receive notifications

Reply:
/cc @procot Do you agree to be the owner?

Reply:
Yes of course :)